packages/http-client-csharp/eng/scripts/Generate.ps1

```diff
@@ -111,6 +110,24 @@ foreach ($directory in $directories) {
         exit $LASTEXITCODE
     }
 
+    # srv-driven contains two separate specs, for two separate clients. We need to generate both.
+    if ($folders.Contains("srv-driven")) {
+        $specFile = Join-Path $directory.FullName "old.tsp"
+        $generationDir = Join-Path $generationDir "v1"
+        if (-not (Test-Path $generationDir)) {
+            New-Item -ItemType Directory -Path $generationDir | Out-Null
+        }
+
+        Write-Host "Generating $subPath v1" -ForegroundColor Cyan
+        # override namespace for "resiliency/srv-driven/old.tsp" (make it different to that from "main.tsp")
+        Invoke (Get-TspCommand $specFile $generationDir $stubbed " --option @typespec/http-client-csharp.namespace=Resiliency.ServiceDriven.V1")
```

Comment:
Why do they need to be in separate namespaces? Do the names conflict otherwise?

Reply:
Yeah having both generated clients in the same namespace results in several types having conflicts.